src/OmniSharp.Abstractions/Options/CodeActionOptions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+namespace OmniSharp.Options
+{
+    public class CodeActionOptions
+    {
+        public string[] LocationPaths { get; set; }
+
+        public IEnumerable<string> GetLocations(IOmniSharpEnvironment env)
+        {
+            if (LocationPaths == null) return Enumerable.Empty<string>();
+
+            var normalizePaths = new HashSet<string>();
+            foreach (var locationPath in LocationPaths)
+            {
+                if (Path.IsPathRooted(locationPath))
```

Comment:
yeah so at the moment, relative always means relative to the folder in which OmniSharp is running. 
I think this is reasonable for most cases, but you are right, when you are editing the global file like `c:\users\filip\.omnisharp\omnisharp.json` it might appear a bit counter intuitive.